crates/sui-source-validation-service/src/lib.rs

```diff
@@ -145,17 +146,19 @@ pub async fn verify_package(
         .await?;
 
     let mut map = SourceLookup::new();
-    let Ok(address) = compiled_package.published_at.as_ref().map(|id| **id) else { bail!("could not resolve published-at field in package manifest")};
```

Comment:
`rustfmt` struggles a lot with `let ... else` formatting so I fixed these up to be more readable